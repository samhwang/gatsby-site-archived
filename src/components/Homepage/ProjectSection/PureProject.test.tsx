/* eslint-disable import/no-extraneous-dependencies */
import { render } from '@testing-library/react';
import PureProject from './PureProject';

describe('Project Section Component', () => {
  it('Should render without crashing and match snapshot', () => {
    const projects = [
      {
        title: 'title',
        description: 'description',
        url: 'https://samhwang.github.io',
      },
    ];
    const tree = renderer.create(<PureProject projects={projects} />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
