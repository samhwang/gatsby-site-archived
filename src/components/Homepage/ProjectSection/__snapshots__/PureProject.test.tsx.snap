// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Project Section Component Should render without crashing 1`] = `
<section
  className="resume-section p-3 p-lg-5 d-flex align-items-center"
  id="projects"
>
  <div
    className="w-100"
  >
    <h2
      className="mb-5"
    >
      Projects
    </h2>
    <ul
      className="fa-ul mb-0"
    >
      <li>
        <i
          className="fa-li fa fa-tasks text-warning"
        />
        <p>
          <a
            href="https://samhwang.github.io"
            onClick={[Function]}
          >
            <strong>
              title:
            </strong>
          </a>
           description
        </p>
      </li>
    </ul>
  </div>
</section>
`;
